src/scss/components/optional/_content-title.scss

```diff
@@ -30,7 +30,7 @@
             margin: 0;
             width: 100%;
             font-weight: $font-weight-base;
-            @include font-size(base--scaleUp, false);
+            @include font-size(body-l, false);
```

Comment:
`_content-title.scss` - again `heading-s` maybe?

Reply:
Have added a TODO to this to make sure this is addressed when we port the component (as it should have a different alias token at that point)